t/par_packer.t

```diff
@@ -0,0 +1,39 @@
+use strict;
+use warnings;
+use Test::More;
+BEGIN { delete $ENV{PKG_CONFIG_PATH} }
+use PkgConfig;
+use Config;
+use File::Temp qw( tempdir );
+
+my $have_pp = eval 'require pp';
+
+plan skip_all => "Test only for MSWin32" unless $^O eq 'MSWin32';
+plan skip_all => "Test only for strawberry MSWin32" unless $Config{myuname} =~ /strawberry-perl/;
+plan skip_all => "Test needs pp utility to be installed" unless $have_pp;
```

Comment:
I prefer to have this run if pp is installed without the user opting-in, as it is here.

However, can we make this `unless eval { require pp }`?  That gets rid of the string eval and we don't need `$have_pp` since the test will be skipped without `pp`.